spring-context/src/main/java/org/springframework/format/datetime/DateTimeFormatAnnotationFormatterFactory.java

```diff
@@ -58,15 +58,15 @@
 
 	@Override
 	public Printer<?> getPrinter(DateTimeFormat annotation, Class<?> fieldType) {
-		return getFormatter(annotation, fieldType);
+		return getFormatter(annotation);
 	}
 
 	@Override
 	public Parser<?> getParser(DateTimeFormat annotation, Class<?> fieldType) {
-		return getFormatter(annotation, fieldType);
+		return getFormatter(annotation);
 	}
 
-	protected Formatter<Date> getFormatter(DateTimeFormat annotation, Class<?> fieldType) {
+	protected Formatter<Date> getFormatter(DateTimeFormat annotation) {
```

Comment:
This is a potential hookpoint where a subclass may use that information. Please revert this.